src/azure-cli/azure/cli/command_modules/hdinsight/commands.py

```diff
@@ -104,3 +104,18 @@ def load_command_table(self, _):
                             client_factory=cf_hdinsight_virtual_machines) as g:
         g.command('list', 'list_hosts')
         g.command('restart', 'restart_hosts', confirmation=True)
+
+    # Autoscale operations
+    with self.command_group('hdinsight autoscale', hdinsight_clusters_sdk, client_factory=cf_hdinsight_clusters) as g:
+        g.custom_command('create', 'create_autoscale')
+        g.custom_command('update', 'update_autoscale')
+        g.custom_command('show', 'show_autoscale')
+        g.custom_command('delete', 'delete_autoscale')
```

Comment:
better to add (..., confirmation=True)

Reply:
added.

---
In reply to: [466797697](https://github.com/Azure/azure-cli/pull/14692#discussion_r466797697) [](ancestors = 466797697)